docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  vitalam-node:
+    image: ghcr.io/digicatapult/vitalam-node:v2.3.0
+    ports:
+      - 30333:30333
+      - 9944:9944
+      - 9933:9933
+    environment:
+      - IDENTITY=dev
+      - WS=true
+      - RPC=true
+      - CORS=all
```

Comment:
I believe we now need to change the `CMD` that is run on the vitalam-node container as we no longer use ENV variables and a `docker-run.sh` file to insert these as arguments.

Reply:
made a bug ticket for this earlier https://digicatapult.atlassian.net/browse/IN-65

Reply:
I don't see why the service template even needs the `vitalam-node`?